diff -ruN ../git/UoI_Lasso_V3/Makefile ./Makefile
--- ../git/UoI_Lasso_V3/Makefile	2019-03-08 20:15:29.345311000 -0800
+++ ./Makefile	2019-05-05 23:40:44.158921000 -0700
@@ -1,18 +1,18 @@
 TARGETS=uoi
-OBJS = distribute-data.o lasso.o bins.o manage-data.o CommandLineOptions.o
+OBJS = distribute-data.o lasso.o bins.o manage-data.o CommandLineOptions.o logspace.o alpha_grid.o
 
 
 CXX=CC #for PC mpic++
 #for PC EIGEN3=-I /usr/local/include/eigen3/
-ADD_FLAGS = #-DCIRCULARDEPENDENCE #additional flags can be added here. 
+ADD_FLAGS = #-DALPHAGRID #-DCIRCULARDEPENDENCE #additional flags can be added here. 
 
-BOOST = -I${BOOST_ROOT}/include
+BOOST = -I$(BOOST_ROOT)/include
 CXXFLAGS=-Wall -g  -O3 -std=c++11 $(EIGEN3) $(BOOST) $(GSL) -xMIC-AVX512 -qopenmp -dynamic -debug inline-debug-info -mkl -std=c++11  -fp-model fast=2
 CC=CC #for PC mpicc
 CCFLAGS=-g  -O3  -xMIC-AVX512 
 #BOOSTFLAGS =  -L/usr/common/software/boost/1.63/intel/mic-knl/lib/ #-lboost_program_options -lboost_filesystem -lboost_system
 
-BOOSTFLAGS =  -L${BOOST_ROOT}/lib/
+BOOSTFLAGS =  -L$(BOOST_ROOT)/lib/
 
 .PHONY: all clean
 
